### ISSUE TITLE
Command line arguments are ignored when a config file is present

### ISSUE THREAD

**Issue by roflolilolmao** [turnkey-commerce/gendal#28]
Since version `1.2.7`, command line arguments are ignored when a config file is present. The following scenarios happen:

- when a config file is present and does not provide a DSN:

```zsh
❯ gendal --version
error: Database DSN must be provided

❯ gendal "postgres://postgres@localhost/some_db"  [...]
error: Database DSN must be provided
```

- when no config file can be found, the following happens:

```zsh
❯ gendal --version
config file not found, using command line arguments
gendal version 1.2.7

❯ gendal "postgres://postgres@localhost/some_db" [...]
config file not found, using command line arguments
[gendal generates according to CLI arguments]
```

- when a config file is present and provides a DSN:

```zsh
❯ gendal --version
[gendal generates using the config file options; no CLI ouput as expected]

❯ gendal --suffix ".a_random_suffix"
[gendal generates but ignores the `suffix` option]
```

**Comment by turnkey-commerce** [comment]
Thanks for the report. I had envisioned that the config file versus command line args was an either/or scenario but obviously you have a use case that requires both. This was a side effect of fixing another issue that caused some config file inputs to be ignored.

**Comment by turnkey-commerce** [comment]
Fixed in version v1.2.8.  Thanks @roflolilolmao  for raising the issue.

**Comment by roflolilolmao** [comment]
Thank you. :-)
